patches/components-sync-engine-syncer.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/components/sync/engine/syncer.cc b/components/sync/engine/syncer.cc
+index c927a5e12fcb709761f6d3c04d89f55f35a902ec..a4a88012ebc33eca8d75979e7cf1b72e093aa02a 100644
+--- a/components/sync/engine/syncer.cc
++++ b/components/sync/engine/syncer.cc
+@@ -188,6 +188,7 @@ SyncerError Syncer::BuildAndPostCommits(const ModelTypeSet& request_types,
+     }
+     nudge_tracker->RecordSuccessfulCommitMessage(
+         commit->GetContributingDataTypes());
++    BRAVE_BUILD_AND_POST_COMMITS_POLLER_CHECK
```

Comment:
please don't forget class name:
`BRAVE_SYNCER_BUILD_AND_POST_COMMITS_POLLER_CHECK`

Reply:
Thanks, fixed a at https://github.com/brave/brave-core/pull/17077/commits/33e48b2f5c8e49e9f218c3f9f3d2ff6579126ef5 

But this seems redundant to me because `components/sync/engine/syncer.cc` has only one class. Is there any practical point except the tradition?

Reply:
> But this seems redundant to me because `components/sync/engine/syncer.cc` has only one class. Is there any practical point except the tradition?

it's a rule that declares a clear baseline for everyone. At any point there might appear another class or a global function with similar name. So I prefer just follow this strictly and don't think about all conflicts that may or may not appear during code evolution.

Reply:
:+1: thanks for the clarification